code/modules/surgery/prosthetic_replacement.dm

```diff
@@ -17,7 +17,11 @@
 
 /datum/surgery_step/add_prosthetic
 	name = "add prosthetic"
-	implements = list(/obj/item/bodypart = 100, /obj/item/organ_storage = 100, /obj/item/chainsaw = 100, /obj/item/melee/synthetic_arm_blade = 100)
+	implements = list(
+		/obj/item/bodypart = 100,
+		/obj/item/organ_storage = 100,
+		/obj/item/chainsaw = 100,
```

Comment:
I think there's a conversation to be had about just removing chainsaw arms here

Reply:
its kinda cool tho